app/assets/javascripts/hooch.js

```diff
@@ -1088,6 +1088,114 @@ var initHooch = function(){
       replaceWith: function($jq_obj){
         this.$sort_element.replaceWith($jq_obj)
       }
+    }),
+    key_code_map: { // borrowed from jresig: https://github.com/jeresig/jquery.hotkeys
+          "backspace": 8,
+          "tab": 9,
+          "return": 10,
+          "return": 13,
```

Comment:
I guess I need an array. bah. I'll make a todo for it, I seriously doubt we'll ever use this again.